core/services/pipeline/common.go

```diff
@@ -310,6 +310,8 @@ const (
 	TaskTypeETHABIDecode     TaskType = "ethabidecode"
 	TaskTypeETHABIDecodeLog  TaskType = "ethabidecodelog"
 	TaskTypeMerge            TaskType = "merge"
+	TaskTypeLength           TaskType = "length"
```

Comment:
You must:
1. Describe these new tasks in our documentation here https://github.com/smartcontractkit/documentation and provide a link to the PR, so both are reviewed together.
2. Mention the new tasks and other changes in CHANGELOG, so QA team knows what changes to test.

Reply:
Done. documentation PR https://github.com/smartcontractkit/documentation/pull/989

Reply:
You added `gasUnlimited` param to eth_call task, that also need to be reflected in the doc.

Reply:
pushed an update to the PR. There were some other parameters missing, i've also added documentation for them, could you verify that once